src/Program.fs

```diff
@@ -1939,7 +1947,20 @@ let createOpenApiClient
         for operation in safeSeq pathInfo.Operations do
             let operationInfo = operation.Value
             if not operationInfo.Deprecated && includeOperation operationInfo config then
-                let parameters = operationParameters operationInfo pathInfo.Parameters config
+
+                let parameters = [
+                    if config.IsCancellable then {
```

Comment:
That's right (and I would highly prefer that) but if I do this

```
                let parameters = [
                    yield! operationParameters operationInfo pathInfo.Parameters config 

                    if config.IsCancellable then {
                        parameterName = "ct"
                        parameterIdent = "ct"
                        required = false
                        parameterType = SynType.Create "System.Threading.CancellationToken"
                        docs = null
                        location = null
                        properties = []
                        style = "none" } ]
```

then I get this at runtime:

![image](https://user-images.githubusercontent.com/8280970/229936194-06ea7d73-38c7-45dd-82bf-6dfe9b793f20.png)

I tried a couple of things but haven't the time to dig too much. Any hints?

Reply:
> Any hints?

Not exactly sure to be honest. This however is very much needed because if the cancellation token is optional as a last parameter with a default value, then it shouldn't be necessary to have separate targets `async-ct` and `task-ct`, Hawaii should simply generate the extra command 

If this is too much to implement, you can go back to the original workaround of using `SendAsync(request, CancellationToken.None)` and let the user override the `HttpClient` or subclass a `HttpMessageHandler` that accepts a `CancellationToken` and overrides the provided `None` ct

Reply:
> If this is too much to implement, you can go back to the original workaround of using SendAsync(request, CancellationToken.None) and let the user override the HttpClient or subclass a HttpMessageHandler that accepts a CancellationToken and overrides the provided None ct

This is hard when parallel request are sent. When the SendAsync override is called, ct is not there. 
ct  must be passed down the chain, or we should serialize the calls and keep ct cached to replace in the override. Not an iedal solution

Reply:
@Zaid-Ajaj   Found and alternative way, see last commit